applications/CompressiblePotentialFlowApplication/tests/stochastic_body_fitted_opt/xmc_parameters.json

```diff
@@ -53,7 +53,7 @@
     "monoCriteriaInpuctDictionary" :{
         "statisticalError": {
             "criteria": "xmc.methodDefs_monoCriterion.criterionFunctions.isLowerThanOrEqualTo",
-            "tolerance": [1.1e-03],
+            "tolerance": [100.0],
```

Comment:
Why this change? If you want to be sure that you do not do more than one iteration, you can change the key `maxNumberIterations`.

Reply:
Ok thanks!